components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -387,12 +393,86 @@
             saveTemplate, null);
     }
 
+    function validateTextForIllegal(fld) {
+        var isValid = doValidateInput(fld, "Provided Service Provider Template name is invalid.");
+        if (isValid) {
+            return true;
+        } else {
```

Comment:
Can we simplify this as below?

 if (isValid) {
            return true;
        }
return false;